tensorflow_datasets/core/features/image_feature_test.py

```diff
@@ -109,6 +109,22 @@ def test_image_shaped(self):
         ],
     )
 
+  def  test_image_dtype(self):
```

Comment:
Please follow the style for consistency:
https://www.tensorflow.org/datasets/add_dataset#5_check_your_code_style

You can run pylint to ensure the file is compliant.
```
./oss_scripts/lint.sh this/file.py 
```